packages/firebase_dynamic_links/lib/src/dynamic_link_components.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of firebase_dynamic_links;
+
+/// The class used for Dynamic Link URL generation.
+///
+/// Supports creation of short and long Dynamic Link URLs. Short URLs will have
+/// a domain and a randomized path. Long URLs will have a domain and a query
+/// that contains all of the Dynamic Link parameters.
+class DynamicLinkComponents {
+  DynamicLinkComponents({
```

Comment:
Dartdoc says this class is used for Dynamic Link URL generation. Dartdoc also implies that instances somehow describe Dynamic Link parameters. The class is called DynamicLinkComponents.

This is mildly confusing terminology...